using_images/s2i_images/ruby.adoc

```diff
@@ -96,6 +96,12 @@ and ruby gem installation. The application assets are only compiled if
 Asset compilation only happens when the application runs in a production
 environment. Therefore, you can use this variable when assets have already been
 compiled.
+
+|`*PUMA_MIN_THREADS*`, `*PUMA_MAX_THREADS*`
+|Indicates the minimum and maximum threads that will be available in link:https://github.com/puma/puma[Puma]'s thread pool.
```

Comment:
Since it's a link to an external site, you don't need "link:" in the markup for the Puma link, as per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites